sdkit/models/models_db/__init__.py

```diff
@@ -1,35 +1,53 @@
+"""
+    Models database
+"""
 import json
 from pathlib import Path
 
 db = None
 index = None
 
+
 def get_models_db():
+    """
+        Returns a dictionary of models info
+    """
+    # TODO: Use context vars instead.
     global db
 
     if db is not None:
         return db
 
     db_path = Path(__file__).parent
     db = {}
-    with open(db_path/'stable_diffusion.json') as f: db['stable-diffusion'] = json.load(f)
-    with open(db_path/'gfpgan.json') as f: db['gfpgan'] = json.load(f)
-    with open(db_path/'realesrgan.json') as f: db['realesrgan'] = json.load(f)
+    with open(db_path/'stable_diffusion.json') as fp:
```

Comment:
This might be coding-style preference, but I'd like to keep these on a single line for readability of the logic.

Reply:
having `with` on the same line as the code run within the context is not usual tbh.